idd/Energy+.idd.in

```diff
@@ -750,6 +750,22 @@ ZoneCapacitanceMultiplier:ResearchSpecial,
        \minimum> 0.0
        \note Used to alter the capacitance of zone air with respect to zone air generic contaminant concentration
 
+HVACSystemRootFindingAlgorithm,
```

Comment:
@lgu1234 Just one comment here.  Minor detail, but I would place this object after Timestep or after ConvergenceLimits.  But maybe it doesn't matter.  The whole order of things in this section of the IDD doesn't make much sense anymore.
@Myoldmopar This one's all yours.  I have not further comments.